### DISCUSSION TITLE
How to Add a color or undeline when I hover a link?

### DISCUSSION THREAD

**Discussion by brodneil** [webstudio-is/webstudio-community#139] [General]
I am very new to webstudio and I am not sure how to Add a color or undeline when I hover a link. 

Is there also a way to directly just edit in html or css?

Thank you.

**Comment by kof** [comment]
Did you miss this entire menu? 
<img width="267" alt="image" src="https://github.com/webstudio-is/webstudio-community/assets/52824/be0c0ffa-d2c8-42b9-9ee5-057376916c2a">

**Reply by brodneil** [reply]
Yes, I did. Thank you so much.
